roles/openshift_master_upgrade/meta/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  author: your name
+  description: OpenShift Master upgrade
+  company: Red Hat, Inc.
+  license: Apache License, Version 2.0
+  min_ansible_version: 2.1
+  platforms:
+  - name: EL
+    versions:
+    - 7
+dependencies:
+- role: openshift_common
```

Comment:
I'm not sure we can rely on variables to be set by a role from a different play.